ios/RNTrackPlayer/RNTrackPlayer.swift

```diff
@@ -21,6 +21,9 @@ public class RNTrackPlayer: RCTEventEmitter, AudioPlayerDelegate {
         return player
     }()
     
+    private var sessionCategory: AVAudioSession.Category = .playback
+    private var sessionCategoryOptions: AVAudioSession.CategoryOptions? = []
```

Comment:
Why does this need to be an optional? As far as I can tell, there's no need for it to be.

Reply:
I had taken out the optional but I wasn't sure if it would affect the different type of `if let` we had written. just realized that we're initializing it so it'll likely never be `nil` haha. my bad. i'll change it.